src/endpoints/models/EventModel.js

```diff
@@ -0,0 +1,47 @@
+export default class EventModel {
```

Comment:
We usually export in a separate line at the end of the file, at least I hope so :P

Reply:
No, we don't and I don't think it enhances CodeQuality significantly :P

Reply:
In every component we're doing it the other way around